src/main/java/com/cloudbees/jenkins/support/actions/SupportObjectAction.java

```diff
@@ -59,12 +59,16 @@ public final T getObject() {
 
     @Override
     public String getUrlName() {
-        return "support";
+        return shouldDisplay() ? "support" : null;
     }
 
     @Override
     public String getIconFileName() {
-        return "/plugin/support-core/images/24x24/support.png";
+        return shouldDisplay() ? "/plugin/support-core/images/24x24/support.png" : null;
+    }
+
+    boolean shouldDisplay() {
+        return Jenkins.get().hasPermission(Jenkins.ADMINISTER);
```

Comment:
Shouldn't you use `SupportAction.CREATE_BUNDLE` instead ?

https://github.com/jenkinsci/support-core-plugin/blob/master/src/main/java/com/cloudbees/jenkins/support/SupportAction.java#L71

Reply:
I reused the same as https://github.com/jenkinsci/support-core-plugin/pull/220/files/747dc40b634af540503515b01efc4ca8a2ded2c0#diff-018a199824d332a24f6b86f5c65bdf37R127

Reply:
This is pointing to your same commit?

Reply:
Yeah sorry link was bad, I meant: https://github.com/jenkinsci/support-core-plugin/blob/747dc40b634af540503515b01efc4ca8a2ded2c0/src/main/java/com/cloudbees/jenkins/support/actions/SupportObjectAction.java#L127

This is an interesting point, because `SupportAction.CREATE_BUNDLE` defines a permission (Download Bundle) but the action I linked checks for ADMINISTER :)

I managed to reproduce the issue:
* have a user with Overall/Read + Job/read + Download Permission
* go to a job page
* click on generate bundle (for the job)
* you won't be allowed to generate the bundle

Question is: is it by design, ie only an admin can retrieve a build directory? Or is it a bug?
If it's a bug, happy to fix it while fixing the action visibility issue.

Reply:
Maybe @Dohbedoh can better illustrate us.